source/tutorials/utilities.rst

```diff
@@ -177,6 +177,78 @@ run, using ``inspect-metadata``, we can learn all about it:
 
    How many IDs are there? How many columns? Are there any categorical columns? Why?
 
+Casting Metadata
+................
+
+In the :doc:`Metadata tutorial <metadata>` we learned about column types and utilizing the 
+``qiime tools cast-metadata`` tool to specifiy column types within a provided metadata file.
+Below we will go through a few scenarios of how this tool can be used, and some common mistakes
+that may come up.
+
+We'll start by first downloading some sample metadata. **Note**: This is the same sample
+metadata used in the **Inspect Metadata** section, so you can skip this step if you have
+already downloaded the ``sample_metadata.tsv`` file from above.
+
+.. download::
+   :url: https://data.qiime2.org/2021.8/tutorials/pd-mice/sample_metadata.tsv
+   :saveas: sample_metadata.tsv
+
+In this example, we will cast the ``days_post_transplant`` column from ``numeric`` to ``categorical``,
+and the ``mouse_id`` column from ``categorical`` to ``numeric``. The rest of the columns contained
+within our metadata will be left as-is.
+
+.. command-block::
+   :stdout:
+
+   qiime tools cast-metadata sample_metadata.tsv \
+   --cast days_post_transplant:categorical --cast mouse_id:numeric
```

Comment:
Can you split this into two separate lines, that will match the general style of these docs, and in general I have found it also helps make the content of a command a bit more clear.

```bash
--cast days_post_transplant:categorical \
--cast mouse_id:numeric
```